root/etc/e-smith/templates/etc/ssh/sshd_config/70Restricted2Sftp

```diff
@@ -0,0 +1,26 @@
+{
+    # restrict groups to sftp, ssh access is refused
+    use strict;
+    use esmith::ConfigDB;
```

Comment:
Please see the other [comment](https://github.com/NethServer/nethserver-openssh/pull/8/files#diff-5f032d38fa07cb287a0402ad33a4730dR12)

```suggestion
```

Reply:
done